corehq/apps/cloudcare/views.py

```diff
@@ -421,6 +423,23 @@ def get(self, request, *args, **kwargs):
         })
 
 
+def get_apps_by_user(domain, user_ids):
+    groups_by_user = get_groups_by_user(domain, user_ids)
+
+    app_access = ApplicationAccess.get_by_domain(domain)
```

Comment:
yes, sorry i meant that the user in ES has its group info, so it'd be easy to add to the dict